assets/i18n/en.json

```diff
@@ -773,8 +773,13 @@
     "I18N_TOPNAV_ADMIN_PAGE": "Admin Page",
     "I18N_TOPNAV_BLOG": "Blog",
     "I18N_TOPNAV_BLOG_DASHBOARD": "Blog Dashboard",
-    "I18N_TOPNAV_CLASSROOM": "Classroom",
-    "I18N_TOPNAV_CLASSROOM_BASIC_MATHS": "Basic Mathematics",
```

Comment:
We will have to remove these from all languages fofr the backend tests in shard 1 to pass.

Reply:
I have not deleted these keys, I've just updated their values